lib/resources/utils.dart

```diff
@@ -62,6 +63,18 @@ class Utils {
 
   static PackageInfo packageInfo = PackageInfo(
       version: 'Unknown', buildNumber: 'Unknown', appName: 'Unknown');
+
+  static bool feedbackEnabled = false;
+
+  static Future<RemoteConfigService> getRemoteConfig() async {
+    try {
+      final remoteConfig = await RemoteConfigService.getInstance();
```

Comment:
Can't this be called directly from main()? Since you added a class for the Remote Config (which is great btw), we shouldn't need to reference it in Utils (this file is more like a place where we put stuff we didn't know where else to fit).